HyperPlay-Gaming/hyperplay-desktop-client
# [UI] Cleanup App and Game Settings

This is the first part of refactoring the Settings.
It removes settings that are broken or not necessary anymore and adjust certain settings depending on the runner since not all of them affects HyperPlay Games

### Changes

- Removed Anti-cheat settings for HyperPlay Games. Reason: Won't work.
- Removed `AltGOGDL` and `AltLegendaryEXE` advanced settings Reason: Not useful
- Removed "Ignore Game Updates" setting for HyperPlay games: Reason, all games should be on the latest version.
- Commented out for now the Disable Controller settings since controller navigation is broken since updating to Mantine-UI.

### How to Test

Check if the settings were properly removed and if the Anti-Cheat Game settings only appears on Epic and GOG Games (should still appear on the Main Game settings on HyperPlay Settings).

---

Use the following Checklist if you have changed something on the Backend or Frontend:
